src/Administration/Resources/app/administration/src/app/component/form/sw-price-field/index.js

```diff
@@ -412,10 +418,12 @@ export default {
         },
 
         onPriceGrossChangeDebounce: debounce(function onPriceGrossChange() {
+            this.$emit('price-calculate', true);
```

Comment:
This is emitting `price-calculate` twice (1st here, 2nd during `convertGrossToNet`)

Reply:
We have several edge cases in the price field which needs to be taken into account. Because the calculation will be done on server side and during the calculation the user shouldn't be able to save e.g. the product. For this we have the `price-calculate` event.

Adding this event inside the debounced method would only emit this after the debounce which shouldn't be the case